migrate/migrations/20170530135027-schedule-rotation.sql

```diff
@@ -11,6 +11,7 @@ INSERT INTO schedules (id, name, description, time_zone)
     SELECT id::UUID, name, description, 'America/Chicago' FROM schedule;
 
 CREATE TYPE enum_rotation_type AS ENUM (
+    'monthly',
```

Comment:
We cannot edit existing migrations, only add new ones.

Migrations will not be applied again (without wiping the DB) and new installations will then not match existing. Add a new value with `ALTER TYPE ... ADD VALUE` with `notransaction` set.